sdk/storage/azure-storage-blob-cryptography/README.md

```diff
@@ -159,4 +159,4 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
 
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-java/sdk/tracing/README.png)
```

Comment:
This is also there in [azure-storage-file-datalake](https://github.com/Azure/azure-sdk-for-java/edit/master/sdk/storage/azure-storage-file-datalake/README.md). You might wanna correct that too.

Reply:
Datalake "README" is currently empty and is still work-in-progress. The package names may change. I will leave that for now as it's not going to GA and will wait for the README to be populated with content.